bin/release

```diff
@@ -4,17 +4,12 @@
 #
 # Assumes bin/prep-release was run and the PR merged.
 #
-# Usage: bin/release VERSION
+# Usage: bin/release
 #
 ###
 set -e
 
-if [ -z "$1" ]; then
-  echo "usage: bin/release VERSION" >&2
-  exit 64
-fi
-
-version=$1
+version=$(< VERSION)
 
 git checkout master
```

Comment:
I guess the simplest thing is to just move the `version=` line to after the `checkout`/`pull`